lib/tukey/data_set/label.rb

```diff
@@ -1,4 +1,4 @@
-require 'ostruct'
+require 'active_support/ordered_options'
```

Comment:
De benchmark van de verschillen tussen openstruct, OrderedOptions en Hash opnemen in de commit message zou hier ook niet verkeerd zijn.